geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaMetadata.scala

```diff
@@ -112,6 +112,7 @@ class KafkaMetadata[T](val config: KafkaDataStoreConfig, val serializer: Metadat
     val props = new Properties()
     props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, config.brokers)
     config.producers.properties.foreach { case (k, v) => props.put(k, v) }
+    config.consumers.properties.foreach { case (k, v) => props.put(k, v) }
```

Comment:
```suggestion
    config.consumers.properties.foreach { case (k, v) => props.put(k, v) }
    config.producers.properties.foreach { case (k, v) => props.put(k, v) }
```

i'd just suggest switching them, so that producer props take precedence (preserving the existing behavior if there are conflicting keys)